_posts/2017-09-26-totw-1.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Tip of the Week #1: <code>string_view</code>"
```

Comment:
What's the display issue here? These are supposed to be displayed as monospaced

Reply:
This one is fine, the <> in StatusOr on 181 is causing problems:
https://abseil.io/tips/

Reply:
Ah, yes, does change the one in StatusOr to <code>StatusOr&lt;gt;</code> solve it ? I'd rather us remain consistent with code in titles.

Reply:
We're already inconsistent and I'd prefer the inconsistency to fixing things manually. Can we accept this automated change and work on fixing the automation as a follow-up?

Reply:
Sure